test/functional/update/boot-file/lines-checked

```diff
@@ -1,5 +1,6 @@
 Update started.
 Preparing to update from 10 to 100
+REGEXP:Error for .*pack-os-core-from-10-to-100.tar download: Response 0 - Couldn't read a file:// file
```

Comment:
Why are all of these necessary all of a sudden, and does your next commit make them unnecessary again?

Reply:
It appears it is still necessary.

Reply:
These errors were occurring before, but there weren't any logs for them. And the parallel download is logging that.

Reply:
hmm, is there a way to ignore errors in the delta pack case? not having a delta pack is not an error since delta packs are optional.